res/TensorFlowLiteRecipes/Part_While_000/test.readme

```diff
@@ -0,0 +1,4 @@
+test.readme of Part_While_000
+
+MAXIMUM and MINIMUM Op exist to make random input to 0
+to make this model loop from 0 to 10.
```

Comment:
I'm not sure it is intended?
```suggestion
MAXIMUM and MINIMUM Op exist to make random input
to make this model loop from 0 to 10.
```

Reply:
> I'm not sure it is intended?

Yup :) Very large random value as input +1 will not change so, WHILE will loop in infinite...
Limitation is to prevent that.